readme.md

```diff
@@ -1,5 +1,113 @@
+
+**Seed** - a project which has been reviewed by the Project Committee and meets the basic Eligibility requirements but does not yet meet the Health requirements. Projects at this level are eligible for some .NET Foundation project services and will be re-evaluated automatically each quarter by the Project Committee to determine if they are ready to graduate to the Member level.
+
+**Member** - a project which satisfies the Eligibility and Health requirements and has been on-boarded as an official .NET Foundation project. Projects at this level can identify themselves publicly as Member Projects and are eligible to obtain access to all .NET Foundation project services. They will continue to be monitored on a monthly basis to ensure they meet the Eligibility and Health criteria.
+
+**Retired** - a project which was once a Member Project but no longer satisfies either the Eligibility or Health requirements, or has submitted a formal request to withdraw from the .NET Foundation.
+
+## Eligibility Criteria
+
+Broken into a variety of functional areas, the eligibility criteria is a checklist which defines if a project is fully aligned with the mission and values of the .NET Foundation.
+
+**Suitability**
+
+- The project operates within moral and ethical standards of the .NET Foundation.
+- The project is aligned with the philosophy and guidelines for collaborative development.
+- The project is built on the .NET platform and/or creates value within the .NET ecosystem.
+- The project solves a large enough problem and has a clear technology scope and focus which provides a reasonable likelihood of success.
+
+**Code**
+
+- The project produces software for distribution to the public at no charge.
+- The project's code is easily discoverable and publicly accessible (preferably on GitHub). 
+- The project contains a build script that can produce deployable artifacts that are identical to the official deployable artifacts, with the exception of code signing (Exception may be granted for strong name keys, though strongly encouraged to be committed. Exception relies on OSS signing being in the build script for public builds).
+- When applicable, project must use reproducible build settings in its toolchain.
+- The project uses [Source Link](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/sourcelink).
+- The project uses either embedded PDBs or publish symbol packages to NuGet (if applicable).
+- The project code signs their artifacts as appropriate.
+
+**Licenses and Copyright**
+
+- The code is offered under a standard, permissive open source license which has been approved by the .NET Foundation.
+- Libraries that are mandatory dependencies of the project are also offered under a standard, permissive open source license which has been approved by the .NET Foundation (exceptions include a dependency on .NET Framework).
+- Committers are bound by a Contributor License Agreement.
```

Comment:
May want to mention the license bots offered by the foundation here.

Reply:
Might be worth using an asterisk to denote requirements that can be met with Foundation services/support. Code Signing is another.